neuraxle/base.py

```diff
@@ -930,9 +1408,12 @@ def fit(self, data_inputs, expected_outputs=None) -> 'NonFittableMixin':
 
 
 class NonTransformableMixin:
-    """A pipeline step that has no effect at all but to return the same data without changes.
+    """
+    A pipeline step that has no effect at all but to return the same data without changes.
```

Comment:
Give example on how a step should inherit from both this and base step and tell that transform method is automatically implemented as changing nothing